usecases/monitoring/prometheus.go

```diff
@@ -714,6 +720,29 @@ func newPrometheusMetrics() *PrometheusMetrics {
 			Help:    "Number of requests required to process an entire (user) batch",
 			Buckets: []float64{1, 2, 5, 10, 100, 1000},
 		}, []string{"vectorizer"}),
+		TokenizerDuration: promauto.NewHistogramVec(prometheus.HistogramOpts{
+			Name:    "tokenizer_duration_seconds",
+			Help:    "Duration of a tokenizer operation",
+			Buckets: []float64{1, 10, 50, 100, 1000, 5000, 10000, 100000, 500000, 1000000, 10000000},
```

Comment:
Does this resolution is sufficient for you? I would expect initialization to take from hundreds of milliseconds to tens seconds? Why do we need bucket for `10000000` which is 115 days?

Reply:
Agree with @faustuzas . I created this bucket to be used for such purposes.
https://github.com/weaviate/weaviate/blob/kavirajk%2Fmake-auto-schema-disabled-runtimeconfig/usecases/monitoring/prometheus.go#L334-L340

Reply:
What if someone runs it on a Raspberry PI, and it takes that long to start? 😂 

It was microseconds, so fixed